Dalamud/Interface/Internal/Windows/Data/Widgets/AddonInspectorWidget.cs

```diff
@@ -1,11 +1,11 @@
-﻿namespace Dalamud.Interface.Internal.Windows.Data.Widgets;
+namespace Dalamud.Interface.Internal.Windows.Data.Widgets;
 
 /// <summary>
 /// Widget for displaying addon inspector.
 /// </summary>
 internal class AddonInspectorWidget : IDataWindowWidget
 {
-    private UiDebug? addonInspector;
+    private UiDebug2.UiDebug2? addonInspector;
```

Comment:
Let's keep both inspectors around for a little bit. Once we can get feedback from devs overall, we can delete the legacy `UiDebug`.